.github/workflows/cache-to-ecr.yml

```diff
@@ -0,0 +1,77 @@
+name: Docker Build Cache Push AWS ECR
+
+on:
+  workflow_dispatch:
+  push:
+    paths-ignore:
+      - 'docs/**'
+      - '**.md**'
+    branches: [main]
+
+jobs:
+  build-and-push-dev-aws:
```

Comment:
Question: A lot of this looks like what's in `.github/workflows/dev-aws-ecr.yml`. Should we at some point reuse the common steps? If so I don't think it needs to be part of this story.

Reply:
Yes I think at some point it makes sense for this to be a reusable workflow so we do not repeat code.